dev/com.ibm.websphere.appserver.features/visibility/public/grpcClient-1.0/io.openliberty.grpcClient-1.0.feature

```diff
@@ -11,12 +11,10 @@ IBM-API-Package: \
 IBM-ShortName: grpcClient-1.0
 Subsystem-Version: 1.0.0
 Subsystem-Name: gRPC Client 1.0
--features=com.ibm.websphere.appserver.servlet-4.0, \
+-features=io.openliberty.grpcClient1.0.internal.ee-8.0; ibm.tolerates:="9.0", \
```

Comment:
This should have com.ibm.websphere.appserver.servlet-4.0; ibm.tolerates:="5.0"

Reply:
Will update.